### PR TITLE
fix: handle failed requests better

### PR DESCRIPTION
**Pull request by kyranjamie**
> [Download the latest build](https://github.com/blockstack/stacks-wallet/actions/runs/488112874)

### CODE REVIEW

File: app/hooks/use-mempool.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ export function useMempool() {
   const mempoolFetcher = useCallback(() => new Api(activeNode.url).getMempoolTransactions(), [
     activeNode,
   ]);
-  const { data: mempoolTxs } = useSWR('mempool', mempoolFetcher, { refreshInterval: 1_500 });
+  const { data: mempoolTxs } = useSWR('mempool', mempoolFetcher, { refreshInterval: 10_000 });
```
Comment (zone117x) [inline app/hooks/use-mempool.ts:29]: 👍

File: app/store/stacking/stacking.reducer.ts [hunk 2]
```diff
@@ -82,8 +82,14 @@ export const stackingSlice = createSlice({
       action: PayloadAction<CoreNodePoxResponse>
     ) => {
       state.initialRequestsComplete.poxInfo = true;
+      state.errors.poxInfo = false;
```
Comment (zone117x) [inline app/store/stacking/stacking.reducer.ts:85]: A lot of state management! Looks okay to me, although not familiar with this code at all

**Review by zone117x — APPROVED** [review]

File: app/store/stacking/stacking.reducer.ts (hunk 2 above)
Comment (kyranjamie) [reply app/store/stacking/stacking.reducer.ts:85]: Yes this is a bit messy state mgmt to be honest, but it has at least been easy to add very explicit behiaviours

File: app/hooks/use-mempool.ts (hunk 1 above)
Comment (aulneau) [reply app/hooks/use-mempool.ts:29]: would recommend using `SWRConfig` at the root and make this happen only once, and then use a constant for it, eg `DEFAULT_POLLING_INTERVAL`

File: app/pages/app.tsx [hunk 3]
```diff
@@ -50,7 +50,7 @@ export const App: FC = ({ children }) => {
 
   useNavigatorOnline({ onReconnect: initAppWithStxAddressInfo });
 
-  useInterval(() => refreshWalletDetailsWithoutLoader(), 1_500);
+  useInterval(() => refreshWalletDetailsWithoutLoader(), 10_000);
```
Comment (aulneau) [inline app/pages/app.tsx:53]: `DEFAULT_POLLING_INTERVAL`

**Review by aulneau — APPROVED** [review]

**Pull request merged**